nevergrad/benchmark/test_experiments.py

```diff
@@ -55,7 +55,7 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
         maker,
         ("mltuning" in name or "anm" in name),
         skip_seed=(name in ["rocket", "images_using_gan"])
-        or any(x in name for x in ["tuning", "image_", "compiler", "anm"]),
+        or any(x in name for x in ["tuning", "image_", "compiler", "anm", "olympus"]),
```

Comment:
is olympus not seedable in some way? If you had to add this here it means the test cases would not be reproducible :(
I see the shift is using the parametrization random state which is seeded so should be good, so that must come from olympus

Reply:
Unfortunately, I have not found any ways to seed the emulators but the surfaces are seedable.

Reply:
too bad :(